include/pinocchio/multibody/joint/joint-composite.hpp

```diff
@@ -464,6 +485,16 @@ namespace pinocchio
       return res;
     }
 
+    int getJointIndex(const std::string & joint_name) const
```

Comment:
```suggestion
    JointIndex getJointId(const std::string & joint_name) const
```
Use the same interface than `ModelTpl`. I don't like the name (getJointId to retrieve the index…) but it's the convention for now.